### PR TITLE
Feat/add announcement to articles page

### LINKED ISSUE #272: Add banner on sidebar for writing challenge
## Context

To try and get people to signup and write we are going to show the temporary writing challenge. 

It will also help us get metrics on click-throughs. 

The first issue is to add a banner before we can track it: 

challenge
Group 63

Screenshot 2023-05-02 at 22 49 07

### PR DESCRIPTION
**Pull request by JohnAllenTech**
# ✨ Codu Pull Request 💻

## Pull Request details:
Added an announcement section to the article page.

Closes #272

## Any Breaking changes:
None

## Associated Screenshots:
    
image

### CODE REVIEW

**Comment by JohnAllenTech** [comment]
Oof looks like something messed up and I am trying to merge the entire codebase in. Ill fix it later

File: styles/globals.css [hunk 1]
```diff
@@ -35,6 +35,15 @@ body {
   }
 }
 
+.announcement-wrapper {
+  margin: 16px 0px;
+  cursor: pointer
+}
+
+.announcement-image {
+  width: 100%
```
Comment (NiallJoeMaher) [inline styles/globals.css]: We can use a Tailwind class name instead of this `w-full`.

File: styles/globals.css [hunk 2]
```diff
@@ -35,6 +35,15 @@ body {
   }
 }
 
+.announcement-wrapper {
+  margin: 16px 0px;
+  cursor: pointer
+}
```
Comment (NiallJoeMaher) [inline styles/globals.css]: We can use tailwind classes for this: `cursor-pointer my-4`

File: pages/articles/index.tsx [hunk 3]
```diff
@@ -175,6 +177,18 @@ const ArticlesPage = () => {
               </section>
             </div>
             <section className="col-span-4 lg:block hidden">
+              <div className="announcement-wrapper">
+                <Image
+                  className="announcement-image"
```
Comment (NiallJoeMaher) [inline pages/articles/index.tsx]: We can use Tailwind instead of CSS here

File: pages/articles/index.tsx [hunk 4]
```diff
@@ -175,6 +177,18 @@ const ArticlesPage = () => {
               </section>
             </div>
             <section className="col-span-4 lg:block hidden">
+              <div className="announcement-wrapper">
```
Comment (NiallJoeMaher) [inline pages/articles/index.tsx]: We can use Tailwind instead of CSS here

**Review by NiallJoeMaher — COMMENTED** [review]
Small things! Great work so far.

File: styles/globals.css (hunk 1 above)
Comment (JohnAllenTech) [reply styles/globals.css]: Ah nice! Never worked with Tailwind before. Its actually kinda nice

**Comment by JohnAllenTech** [comment]
@NiallJoeMaher  should be sorted now

**Review by NiallJoeMaher — APPROVED** [review]

**Pull request merged**